build/Build.cs

```diff
@@ -32,12 +32,14 @@ class Build : NukeBuild
 
     static AbsolutePath SourceDirectory => RootDirectory / "source";
     static AbsolutePath ArtifactsDirectory => RootDirectory / "artifacts";
+    static AbsolutePath TestResultsDirectory => RootDirectory / "TestResults";
```

Comment:
Standardising `TestResults` to have consistent unit test report path for GHA workflow